src/navigation/MainTabNavigator.tsx

```diff
@@ -32,8 +32,7 @@ HomeStack.navigationOptions = {
 
 const CreateStack = createStackNavigator(
   {
-    //Links: CreateScreen,
-    Links: OnboardingScreen,
+    Links: CreateScreen,
```

Comment:
mixed up parts of this file. I meant "Links" should be changed so like Create: CreateScreen. And the above Onboarding can just be Onboarding: OnboardingScreen like it was before.